noxfiles/run_infrastructure.py

```diff
@@ -77,7 +77,7 @@
 EXTERNAL_DATASTORES = list(EXTERNAL_DATASTORE_CONFIG.keys())
 ALL_DATASTORES = DOCKERFILE_DATASTORES + EXTERNAL_DATASTORES
 OPS_TEST_DIR = "tests/ops/"
-API_TEST_DIR = f"{OPS_TEST_DIR}api/"
+API_TEST_DIRS = [f"{OPS_TEST_DIR}api/", "tests/api/v1/endpoints/"]
```

Comment:
This is part of the test directory cleanup. Adding this new test to `tests/api/v1/endpoints` instead of the ops directory